src/Compilers/Core/Portable/Syntax/SyntaxNodeOrTokenList.cs

```diff
@@ -159,7 +145,7 @@ public TextSpan Span
         /// </returns>
         public override string ToString()
         {
-            return _node != null ? _node.ToString() : string.Empty;
+            return _node?.ToString() ?? string.Empty;
```

Comment:
And therefore we don't mind adding code to check if it is null? I think the previous code was better.